samples/indexabletypes.d.ts

```diff
@@ -0,0 +1,12 @@
+// extract example from https://www.typescriptlang.org/docs/handbook/interfaces.html#indexable-types
+interface NumberDictionary {
+    [index: string]: number;
+}
+interface ReadonlyStringArray {
+    [index: number]: string;
```

Comment:
`ReadonlyStringArray` should have the `readonly` modifier. (the example you cite has it, and it makes sense)

Reply:
Oops.
Test has been cleaned up, including EOL at the end of file.